server/firestore/firestore.indexes.json

```diff
@@ -0,0 +1,26 @@
+{
+  // Example:
```

Comment:
I don't think you can have comments in a .json file

Reply:
Definitely can't. Although the firebase json pre-processor allows it. I can pull them out if you prefer